File: mbv_open_api_sdk_csharp/mbv_open_api_sdk_csharp/com/mbv/open/api/sdk/response/BaseResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace com.mbv.open.api.sdk.response
{
    public class BaseResponse
    {

        public string Url { get; set; }

        [JsonProperty("resp_code")]
        public string RespCode { get; set; }

        [JsonProperty("resp_info")]
        public string RespInfo { get; set; }

    }
}

File: mbv_open_api_sdk/src/main/java/com/mbv/open/api/sdk/response/BaseResponse.java
package com.mbv.open.api.sdk.response;

import com.fasterxml.jackson.annotation.JsonProperty;

public class BaseResponse{

	  private String url;
	  @JsonProperty("resp_code")
		private String respCode;
		
		@JsonProperty("resp_info")
		private String respInfo;

		public String getRespCode() {
			return respCode;
		}

		public void setRespCode(String respCode) {
			this.respCode = respCode;
		}

		public String getRespInfo() {
			return respInfo;
		}

		public void setRespInfo(String respInfo) {
			this.respInfo = respInfo;
		}
	  
	  public String getUrl()
	  {
	    return this.url;
	  }
	  
	  public void setUrl(String url)
	  {
	    this.url = url;
	  }

}
